dbt/seeds/etl_full_row_counts.csv

```diff
@@ -2543,7 +2543,7 @@ out_eia__yearly_utilities,2021,7255
 out_eia__yearly_utilities,2022,7526
 out_eia__yearly_utilities,2023,7806
 out_eia__yearly_utilities,2024,8254
-out_eia__yearly_utilities,2025,7372
+out_eia__yearly_utilities,2025,7366
```

Comment:
Does it make sense that this number goes down? Seems like 22 new utilities went in?